src/Refactoring-Core/RBRemoveInstanceVariableRefactoring.class.st

```diff
@@ -58,6 +60,8 @@ RBRemoveInstanceVariableRefactoring >> generateChanges [
 			self breakingChangePreconditions errorString ].
 
 	self privateTransform.
+	self flag: #todo.
+	"the method comment does not indicte this. May be we can add self changes but to all the generateChanges methods. Be consistent. "
 	^ self changes
```

Comment:
@Ducasse I agree with this. I was looking of usages of `generateChanges`:
- half of the test are using `refactoring model changes changes` immediately after calling `generateChanges`, so it might be good for this
- another half are not relying on them, but we can ignore the return value in that situation
- in Drivers we almost always need `changes` after calling `generateChanges`
- in Refactorings/Transformation that don't have the driver, we usually don't need `changes` after calling `generateChanges`

So I think it makes sense to return the changes and be consistent everywhere, even though in some places they are not required, but can be safely ignored.